projects/gnomad-api/src/schema/datasets/gnomad_r2_1/fetchGnomadVariantDetails.js

```diff
@@ -25,7 +25,7 @@ const formatPopulations = variantData => [
       id: `${popId.toUpperCase()}_${subPopId.toUpperCase()}`,
       ac: (variantData.AC_adj[popId] || {})[subPopId] || 0,
       an: (variantData.AN_adj[popId] || {})[subPopId] || 0,
-      ac_hemi: null,
+      ac_hemi: 0,
```

Comment:
This should stay `null`. gnomAD v2 doesn't have subpopulations broken down by sex, so the number of hemizygotes is unknown, not necessarily 0.

This is used by the PopulationsTable component to show a "-" instead of a number.
https://github.com/macarthur-lab/gnomadjs/blob/269157843cdbbec9b7e140151819f9be2e3dc85f/projects/gnomad/src/client/VariantPage/PopulationsTable.js#L263

Reply:
I see. Let me revert that